src/index.html

```diff
@@ -22,6 +22,395 @@
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#apple"
+              class="nav__link nav__link--is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#samsung"
+              class="nav__link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#smartphones"
+              class="nav__link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#laptops"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#gadjets"
+              class="nav__link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#tablets"
+              class="nav__link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#photo"
+              class="nav__link"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#video"
+              class="nav__link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+
+        <button class="nav__menu">
+          <img
+            src="./icons/burger.svg"
+            alt="menu"
+          />
+        </button>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div
+        class="product-card"
+        data-qa="card"
+      >
+        <img
+          src="./images/imac.jpeg"
+          alt="IMac 27 inch Retina 5K"
+          class="product-card__image"
+        />
+        <h2 class="product-card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="product-card__code">Product code: 195434</p>
+
+        <div class="product-card__rating">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="product-card__reviews">Reviews: 5</p>
+        </div>
+
+        <div class="product-card__price">
+          <p class="product-card__label">Price:</p>
+
+          <p class="product-card__cost">$2,199</p>
+        </div>
+
+        <a
+          href="#"
+          class="product-card__button"
+          data-qa="card-hover"
+        >
+          Buy
+        </a>
+      </div>
+      <div class="product-card">
```

Comment:
Add the blank line between the cards
```suggestion
      </div>
      
      <div class="product-card">
```